homeassistant/components/rollershutter/zwave.py

```diff
@@ -0,0 +1,95 @@
+"""
+Support for Zwave roller shutter components.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/rollershutter.zwave/
+"""
+# Because we do not compile openzwave on CI
+# pylint: disable=import-error
+import logging
+from homeassistant.components.rollershutter import DOMAIN
+from homeassistant.components.zwave import ZWaveDeviceEntity
+from homeassistant.components import zwave
+from homeassistant.components.rollershutter import RollershutterDevice
+
+COMMAND_CLASS_SWITCH_MULTILEVEL = 0x26  # 38
+COMMAND_CLASS_SWITCH_BINARY = 0x25  # 37
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Find and return Z-Wave roller shutters."""
+    if discovery_info is None or zwave.NETWORK is None:
+        return
+
+    node = zwave.NETWORK.nodes[discovery_info[zwave.ATTR_NODE_ID]]
+    value = node.values[discovery_info[zwave.ATTR_VALUE_ID]]
+
+    if value.command_class != zwave.COMMAND_CLASS_SWITCH_MULTILEVEL:
+        return
+    if value.index != 1:
+        return
+
+    value.set_change_verified(False)
+    add_devices([ZwaveRollershutter(value)])
+
+
+class ZwaveRollershutter(zwave.ZWaveDeviceEntity, RollershutterDevice):
+    """Representation of an Zwave roller shutter."""
+
+    def __init__(self, value):
+        """Initialize the zwave rollershutter."""
+        from openzwave.network import ZWaveNetwork
+        from pydispatch import dispatcher
+        ZWaveDeviceEntity.__init__(self, value, DOMAIN)
+        self._node = value.node
+        dispatcher.connect(
+            self.value_changed, ZWaveNetwork.SIGNAL_VALUE_CHANGED)
+
+    def value_changed(self, value):
+        """Called when a value has changed on the network."""
+        if self._value.node == value.node:
+            self.update_ha_state(True)
+            _LOGGER.debug("Value changed on network %s", value)
+
+    @property
+    def name(self):
```

Comment:
Just to get rid of the index name of the recognized value. Int this case, the rollershutterdevice would be named: `Fibaro system FGRM222 Roller Shutter Controller Open <node number>`
With this mod, we get: `Fibaro system FGRM222 Roller Shutter Controller`
If this is unwanted, I will remove it.